modules/system/assets/ui/js/jquery.debouncedresize.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * debouncedresize: special jQuery event that happens once after a window resize
+ * https://github.com/louisremi/jquery-smartresize
```

Comment:
This should be under `system/assets/ui/vendor/jquery.debouncedresize.js`

Reply:
p.s. Going to change your suggestion to this:

`system/assets/ui/vendor/jquery-smartresize/jquery.debouncedresize.js`

Just to keep things tidy.